Update pointer to RS including refactoring input field schema.

syoung-smallwisdom opened the pull request:
FYI: @dephillipsmichael

Erin-Mounts commented on BridgeApp/BridgeApp/Data Tracking/Model/SBARemoveTrackedItemStepObject.swift:
```diff
@@ -0,0 +1,108 @@
+    ///        {
+    ///            "identifier": "foo",
+    ///            "type": "selection",
+    ///            "title": "Are you sure you want to remove the items",
```
should there be a question mark after 'items'?

dephillipsmichael replied:
This is a bad example on my part. I'll fix it in the branch I'm sending out soon.

Erin-Mounts commented on BridgeApp/BridgeApp/Data Tracking/Model/SBATrackedItemRemindersStepObject.swift:
```diff
@@ -0,0 +1,173 @@
+        subclassCopy.noReminderSetText = self.noReminderSetText
+    }
+    
+    /// Returns the reminder choice step
```
.

Erin-Mounts commented on BridgeApp/BridgeApp/Data Tracking/Model/SBATrackedItemRemindersStepObject.swift:
```diff
@@ -0,0 +1,173 @@
+        }
+    }
+    
+    /// - parameter result: The most recent result
```
there don't actually appear to be any parameters...

dephillipsmichael commented on BridgeApp/BridgeApp/Data Tracking/iOS/SBARemoveTrackedItemStepViewController.swift:
```diff
@@ -62,6 +62,7 @@ open class SBARemoveTrackedItemStepViewController: RSDStepViewController {
         if let removeStep = self.removeTrackedItemStep,
             let title = removeStep.title {
             if let underlinedSegment = removeStep.underlinedTitleSegment {
+                // TODO: syoung 06/23/2018 Refactor to use HTML for underlining and to use a placeholder such as "%1$@" for range replacement.
```
This sounds promising.  Is there an example of this in the SDK?

syoung-smallwisdom replied:
No, I think that there might be an issue to include support for HTML "attributed text" tags but its not a feature that is currently supported by the Research framework.

syoung-smallwisdom commented:
@Erin-Mounts 
This is code that @dephillipsmichael  is currently working on cleaning up. I just moved some of it and fixed the references to deprecated ui hints and "dataType" fields.
